### ISSUE TITLE
Transport [Communication] terminated as INFO in logs

### ISSUE THREAD

**Issue by reagere** [GoogleCloudPlatform/cloud-bigtable-client#649]
Hello,

From 0.2.3-SNAPSHOT, commit 626ad3f9f6826b534f690a4c6161843a1a3e76d8
Tue Jan 26 10:51:43 2016 -0500

After 2 scans (20 workers on 4 VMs of 1 core), we experienced transport terminated in the next 3 scans.

We scan all the data without filters.

```
Jan 28, 2016 8:56:54 AM com.google.bigtable.repackaged.io.grpc.internal.TransportSet$TransportListener transportShutdown
INFO: Transport com.google.bigtable.repackaged.io.grpc.netty.NettyClientTransport@340faa6d(bigtable.googleapis.com/108.177.15.219:443) for bigtable.googleapis.com/108.177.15.219:443 is being shutdown
Jan 28, 2016 8:56:54 AM com.google.bigtable.repackaged.io.grpc.internal.TransportSet$TransportListener transportTerminated
INFO: Transport com.google.bigtable.repackaged.io.grpc.netty.NettyClientTransport@340faa6d(bigtable.googleapis.com/108.177.15.219:443) for bigtable.googleapis.com/108.177.15.219:443 is terminated

Jan 28, 2016 9:03:43 AM com.google.bigtable.repackaged.io.grpc.internal.TransportSet$TransportListener transportShutdown
INFO: Transport com.google.bigtable.repackaged.io.grpc.netty.NettyClientTransport@48cef87e(bigtable.googleapis.com/108.177.15.219:443) for bigtable.googleapis.com/108.177.15.219:443 is being shutdown
Jan 28, 2016 9:03:48 AM com.google.bigtable.repackaged.io.grpc.internal.TransportSet$TransportListener transportTerminated
INFO: Transport com.google.bigtable.repackaged.io.grpc.netty.NettyClientTransport@48cef87e(bigtable.googleapis.com/108.177.15.219:443) for bigtable.googleapis.com/108.177.15.219:443 is terminated
[INFO] 28-01-2016 09:03:48,131 com.google.cloud.bigtable.grpc.scanner.ResumingStreamingResultScanner info - Reissuing scan after receiving error with status: UNAVAILABLE.
```

Is this critical, and the recovery almost immediate?

I noticed an increase of time on 2 scans that has this kind of communication terminated (about 145s). For the third one, the increase is less significant (about 20s)

**Comment by sduskis** [comment]
I think that #652 will fix this problem.  We intentionally restarted connections ourselves due to a problem that happened in the past in server side behavior, but doesn't seem to be happening anymore.  We're in the process of release the 0.2.3 client, so I didn't want to merge #652 yet, since it may possibly have unanticipated side affects.

**Comment by sduskis** [comment]
We closed #652.  We haven't heard of additional issues.  Closing for now.  Please reopen if you're still seeing the issue.
